build/buildpipeline/osx.groovy

```diff
@@ -5,6 +5,8 @@ simpleNode('OSX10.12','latest') {
         checkout scm
     }
     stage ('Build') {
-        sh "./build.sh --ci /p:Configuration=${params.Configuration}"
+        def logFolder = getLogFolder()
+        def environment = "ASPNETCORE_TEST_LOG_DIR=${WORKSPACE}\\${logFolder}"
+        sh "${environment} ./build.sh --ci /p:Configuration=${params.Configuration}"
```

Comment:
As long as it works. On Windows the whitespace resulting in the directory having a trailing whitespace which the `getLogFolder()` archiver does not support.